internal/appland/client.go

```diff
@@ -84,9 +84,20 @@ func (mapset *MapSet) SetOrganization(org string) *MapSet {
 	return mapset
 }
 
+func (mapset *MapSet) SetBranch(branch string) *MapSet {
+	if mapset.Branch != "" && mapset.Branch != branch {
+		fmt.Fprintf(os.Stderr, "warn: current branch differs from override (%s != %s)", mapset.Branch, branch)
+	}
+
+	mapset.Branch = branch
+	return mapset
+}
```

Comment:
@kgilpin @apotterri 

Updated to always change the branch. If we've detected something different, display a warning. This shouldn't be used outside of a CI/CD environment anyway.

Reply:
Looks good, thanks.